packages/components/psammead-amp-geo/src/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eea: [
```

Comment:
Should this component not be `@bbc/psammead-amp-geo-eea`? As it is specific to `EEA` only. Alternative could be 
```
	import { AmpGeoEEA } from '@bbc/psammead-amp-geo';
```

Reply:
We do not have any Product requirements to change geo location - this component enables the list of countries to be used with reference to the key `eea`. We could add more keys in future. For the sake of an example `asia`, with Asian country codes, and that would still be valid and would not be a breaking change - only an additional definition.